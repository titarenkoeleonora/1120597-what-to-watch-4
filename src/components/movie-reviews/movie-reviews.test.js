import React from "react";
import renderer from "react-test-renderer";
import MovieReviews from "./movie-reviews";

const testReviews = [
  {
    id: 1,
    user: `Kate Muir`,
    date: `December 24, 2016`,
    rating: 8.9,
    comment: `Discerning travellers and Wes Anderson fans will luxuriate in the glorious Mittel-European kitsch of one of the director's funniest and most exquisitely designed movies in years`,
  },
  {
    id: 2,
    user: `Bill Goodykoontz`,
    date: `November 18, 2015`,
    rating: 8.0,
    comment: `Anderson's films are too precious for some, but for those of us willing to lose ourselves in them, they're a delight. "The Grand Budapest Hotel" is no different, except that he has added a hint of gravitas to the mix, improving the recipe.`,
  },
];

it(`Correctlt render`, () => {
  const tree = renderer
    .create(
        <MovieReviews
          reviews={testReviews}
        />
    );

  expect(tree).toMatchSnapshot();
});
