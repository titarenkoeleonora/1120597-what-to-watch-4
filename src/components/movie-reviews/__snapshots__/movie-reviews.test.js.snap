// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Correctlt render 1`] = `
<div
  className="movie-card__reviews movie-card__row"
>
  <div
    className="movie-card__reviews-col"
  >
    <div
      className="review"
    >
      <blockquote
        className="review__quote"
      >
        <p
          className="review__text"
        />
        <footer
          className="review__details"
        >
          <cite
            className="review__author"
          />
          <time
            className="review__date"
            dateTime="2016-12-24"
          />
        </footer>
      </blockquote>
      <div
        className="review__rating"
      />
    </div>
  </div>
</div>
`;
